### DISCUSSION TITLE
Lack of documentation for `deepFreezeFn`

### DISCUSSION THREAD

**Discussion by piotrgredowski** [salesforce/akita#632] [Q&A]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[x] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior
Docstring of [`deepFreezeFn`](https://github.com/datorama/akita/blob/ea131a5cb3bd5ece8a00f413846a59a19fb1841e/docs/docs/config.mdx) doesn't contain any valuable information - only function definition `function customDeepFreeze(state) { return freezedState; }`. Should it be like that?

## Expected behavior
More explanation of how `deepFreezeFn` can be used.

## Minimal reproduction of the problem with instructions

For bug reports please provide the _STEPS TO REPRODUCE_ and if possible a _MINIMAL DEMO_ of the problem via
https://stackblitz.com or similar (you can use this template as a starting point: https://stackblitz.com/github/NetanelBasal/akita-issue-seed).

## What is the motivation / use case for changing the behavior?
Better documentation :)

## Environment

<pre><code>
Angular version: X.Y.Z

Browser:
- [x] Chrome (desktop) version XX
- [x] Chrome (Android) version XX
- [x] Chrome (iOS) version XX
- [x] Firefox version XX
- [x] Safari (desktop) version XX
- [x] Safari (iOS) version XX
- [x] IE version XX
- [x] Edge version XX
 
For Tooling issues:
- Node version: XX  
- Platform:  

Others:

</code></pre>

**Comment by NetanelBasal** [comment]
What is missing? You get the state, and you need to return a frozen version of it.
